src/app/zippy/zippy.component.ts

```diff
@@ -38,6 +33,11 @@ export class ZippyComponent {
   visible = false;
   updatedAsync = false;
 
+  @HostListener('keyup.esc')
+  onEsc() {
+    this.toggle();
+  }
```

Comment:
linter complained here that method definitions are supposed to be after properties.